provisioning/roles/gulp/templates/webpack.config.js

```diff
@@ -30,9 +30,9 @@ module.exports = function(production) {
         {
           loader: 'babel',
           exclude: /(node_modules|bower_components)/,
-          test: /\.jsx?$/,
+          test: /\.js$/,
           query: {
-            presets: ['es2016', 'react']
+            presets: ['es2015']
```

Comment:
You might want to keep the part about transcoding jsx files using the `react` transpiler separately from this one.

I agree we were missing something regarding transpiling pure JS files, but we lose features by correcting it so.

Also, may I ask why es2015 instead of 2016 ?